contracts/Comptroller/Diamond/facets/RewardFacet.sol

```diff
@@ -134,6 +134,32 @@ contract RewardFacet is IRewardFacet, XVSRewardsHelper {
         return 0x151B1e2635A717bcDc836ECd6FbB62B674FE3E1D;
     }
 
+    /**
+     * @notice Seize XVS rewards allocated to holders
+     * @param holders Addresses of the XVS holders
+     * @param recipient Address of the XVS token recipient
+     */
+    function seizeVenus(address[] calldata holders, address recipient) external returns (uint256) {
+        ensureAllowed("seizeVenus(address[],address)");
+
+        uint256 holdersLength = holders.length;
+        uint256 totalHoldings;
+
+        updateAndDistributeRewards(holders, allMarkets, true, true);
+        for (uint256 j; j < holdersLength; ++j) {
+            address holder = holders[j];
+            totalHoldings += venusAccrued[holder];
+            delete venusAccrued[holder];
```

Comment:
Should we emit an event with the seized amount and the holder, per holder?

Reply:
Added https://github.com/VenusProtocol/venus-protocol/pull/417/commits/a63c9c9a72474a5f8aa1f4f79b1bc13514e830cf